ubuntu_bridge_network_configuration.sh

```diff
@@ -10,16 +10,23 @@ source lib/common.sh
 if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
      # Adding an IP address in the libvirt definition for this network results in
      # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
-     # the IP address here
+     # the IP address here.
+     # Create a veth iterface peer.
+     sudo ip link add "$PRO_IF1" type veth peer name "$PRO_IF2"
+     # Create provisioning bridge.
      sudo brctl addbr provisioning
      # sudo ifconfig provisioning 172.22.0.1 netmask 255.255.255.0 up
      # Use ip command. ifconfig commands are deprecated now.
      sudo ip addr add dev provisioning "$PROVISIONING_IP"/"$PROVISIONING_CIDR"
      sudo ip link set provisioning up
 
      # Need to pass the provision interface for bare metal
-     if [ "$PRO_IF" ]; then
+     if [ "$PRO_IF1" && "$PRO_IF2" ]; then
```

Comment:
this should be either $PRO_IF1 **OR** $PRO_IF2 ?

Reply:
Fixed.